.sass-lint.yml

```diff
@@ -0,0 +1,130 @@
+options:
+  merge-default-rules: false
+
+files:
+  include: 'indico/htdocs/sass/**/*.scss'
+  ignore: 'indico/htdocs/sass/lib/**/*'
+
+rules:
+  attribute-quotes: 1
+  bem-depth: 0
+  border-zero:
+    - 1
+    - convention: none
+  brace-style:
+    - 2
+    - style: 1tbs
+      allow-single-line: false
+  class-name-format:
+    - 1
+    - convention: hyphenatedlowercase
+      allow-leading-underscore: false
+  clean-import-paths: 2
+  declarations-before-nesting: 1
+  empty-args:
+    - 1
+    - include: true
+  empty-line-between-blocks: 1
+  extends-before-declarations: 2
+  extends-before-mixins: 1
+  final-newline: 2
+  force-attribute-nesting: 0
+  force-element-nesting: 0
+  force-pseudo-nesting: 0
+  function-name-format:
+    - 2
+    - convention: hyphenatedlowercase
+      allow-leading-underscore: true
+  hex-length:
+    - 1
+    - style: short
+  hex-notation:
+    - 1
+    - style: lowercase
+  id-name-format:
+    - 1
+    - convention: hyphenatedlowercase
+      allow-leading-underscore: false
+  indentation:
+    - 2
+    - size: 4
+  leading-zero:
+    - 2
+    - include: true
+  max-file-line-count:
+    - 1
+    - length: 750
+  max-line-length:
+    - 2
+    - length: 120
+  mixin-name-format:
+    - 2
+    - convention: hyphenatedlowercase
+      allow-leading-underscore: true
+  mixins-before-declarations: 0
```

Comment:
Setting it to warning. Error is too much as there could be cases where you do want a mixin later in case you want a more generic declaration first (e.g. a mixin that sets margin-bottom but you first want to zero all other margins with `margin: 0`).